src/filters/multipart.rs

```diff
@@ -131,17 +123,18 @@ impl Stream for FormData {
 impl Part {
     /// Get the name of this part.
     pub fn name(&self) -> &str {
-        &self.headers.name
+        &self.part.name().unwrap_or("not-set")
```

Comment:
Multipart fields exist without a name?

Reply:
Hey @seanmonstar, on the code there it points to the docs here https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition which says:

```
Only the value form-data, as well as the optional directive name and filename, can be used in the HTTP context.
```

Reply:
The subheading https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#as_a_header_for_a_multipart_body points out:

> The first directive is always form-data, and the header _must_ also include a name parameter to identify the relevant field.

Reply:
True.
Your concern is that if the library is not following the RFC or tat I should put a default that makes more sense on that?

Reply:
Either way, I will open an issue there to try to clarify. I am not sure if it is optional because it is used in both to represent both the header of HTTP and the boundary

Reply:
Sorry that I didn't explain more. I wasn't sure myself. So, I'm thinking that perhaps even if `multer` doesn't think so yet, we would treat a lack of a name as an error, and thus if the name does exist, we can just use `self.part.name().expect("checked for name previously")`. What do you think?

Reply:
all good but, the panic is ok there?

Reply:
Oh, what I was thinking is that constructing a `Part` would require us to check for a `name`, and if it doesn't exist, then `next_field()` would return an `Error`. We wouldn't want an unchecked panic that a client could trigger, no.